README.md

```diff
@@ -72,3 +72,53 @@ REACT_APP_TORNJAK_MANAGER=true npm start
 ```
 
 In this view, there is an additional navigation bar tab titled "Manage Servers" where you may register Tornjak agents.  
+
+### Running Tornjak With Auth
```

Comment:
Maybe we should structure this section like following: 
### Running Tornajk locally
#### User Managment Enabled
....
#### User Managment Disabled
...
### Running Tornjak Remotely, On the Cloud
... 
(including all the required env. variables)
### Build Tornjak Front-end Image
(please specify that `CONTAINER_TAG_FRONTEND` must point at *THEIR* directory, since tsidentity` can be used for pulling only, but not pushing)

Reply:
This is all for Dev, structured it the way you recommended. We should have a doc for cloud deployment though separately and we can link it here.